BiometricsFingerprint.cpp

```diff
@@ -297,7 +306,8 @@ Return<RequestStatus> BiometricsFingerprint::authenticate(uint64_t operation_id,
     r = fpc_set_auth_challenge(sdev->fpc, operation_id);
     if (r < 0) {
         ALOGE("%s: Error setting auth challenge to %ju. r=0x%08X",__func__, operation_id, r);
-        return ErrorFilter(-1);
+
+        return ErrorFilter(-11);
```

Comment:
just remove the errorfilter - it makes no sense.
Return RequestStatus::SYS_EAGAIN directly instead.

Reply:
I agree, I did it like that because I didn't really know why it was done that way, originally.